calabash/Classes/Utils/LPDevice.m

```diff
@@ -291,6 +292,8 @@ - (NSDictionary *) screenDimensions {
                         @"width" : @(size.width),
                         @"scale" : @(scale),
                         @"bounds" : boundsDict,
+                        @"bounds_portrait_height" : @(screenBoundsHeight),
```

Comment:
does it make sense to also log the nativeBounds?

Reply:
What are the native bounds?

Reply:
https://developer.apple.com/documentation/uikit/uiscreen/1617810-nativebounds

Reply:
Added.